src/lib/hooks/useLogin.ts

```diff
@@ -0,0 +1,6 @@
+import { useLoginStore } from "@/lib/stores/useLoginStore";
+
+export function useLogin() {
```

Comment:
what's the point? zustand ne sikim bilmiyom da once onu yapip sonra normal hook'a cevirmissin kral